hops/tests/test_featurestore.py

```diff
@@ -578,36 +578,37 @@ def test_do_prepare_stats_settings(self, sample_metadata):
         assert result["num_clusters"] == 10
         assert result["corr_method"] == "spearman"
 
-    def test_validate_metadata(self):
+    def test_validate_metadata(self, sample_metadata):
```

Comment:
Could you please add a test to check the behaviour when passing capital letters names?

Reply:
Done. Added test for upper case feature group name and feature name, as well as empty names.